Zappy_IA_Report/DTC/Video/Tutorial1.py

```
@@ -0,0 +1,25 @@
+import pandas as pd 
+from sklearn.model_selection import train_test_split
+
+df = pd.read_csv('500hits.csv', encoding = 'latin-1')
```

Flake8 (7.2.0)

[error] unexpected spaces around keyword / parameter equals

(E251)